manifests/init.pp

```diff
@@ -107,7 +108,7 @@
   String                          $smtp_listen         = '127.0.0.1',   # postfix_smtp_listen
   Boolean                         $use_amavisd         = false,         # postfix_use_amavisd
   Boolean                         $use_dovecot_lda     = false,         # postfix_use_dovecot_lda
-  Boolean                         $use_schleuder       = false,         # postfix_use_schleuder
+  Variant[Integer[2, 2], Boolean] $use_schleuder       = false,         # postfix_use_schleuder
```

Comment:
Maybe we could also allow `3`  as a value?

Reply:
Sure, done.